cloud/deploys/dev_azure/README.md

```diff
@@ -3,9 +3,37 @@ This is a deploy directory that allows individuals to create a deploy of the
 app. We intentionally don't use a shared state with this terraform directory
 to allow for everyone to have their own app. 
 
-The process to deploy is similar
-to the staging process but we do not have to do the shared state set up. It 
-should be `terraform init` and then `terraform apply`
+# Running the dev deploy
+## Copy the civiform_config.example.sh 
+Copy the civiform_config.example.sh into civiform_config.sh and
+ change the required variables. 
+
+## Setup Login Radius For Local Development
+Go to [Login Radius Dashboard](https://dashboard.loginradius.com/) and click
```

Comment:
If someone is setting up loginradius from scratch, is there additional setup they'll need to do?